src/mapping/epoch-action.ts

```diff
@@ -10,3 +11,30 @@ export function handleEpochUpdateRewards(
   const rewards = BigDecimal.fromString(data.get('rewards')!.toString());
   updatePrice(event, method, receipt, rewards, BigDecimal.zero());
 }
+
+export function handleEpochUnstakeSuccess(
+  data: TypedMap<string, JSONValue>,
+  receipt: near.ReceiptWithOutcome
+): void {
+  const amount = BigInt.fromString(data.get('amount')!.toString());
+
+  const epochId = receipt.block.header.epochId.toBase58();
+  const validatorId = data.get('validator_id')!.toString();
+  const id = epochId + '#' + validatorId;
+
+  const entity = ValidatorEpochInfo.load(id);
+
+  if (!entity) {
+    const entity = new ValidatorEpochInfo(id);
+
+    entity.epochId = epochId;
+    entity.validatorId = validatorId;
+    entity.epochUnstakedAmount = amount;
+
+    entity.save();
+  } else {
+    entity.epochUnstakedAmount = entity.epochUnstakedAmount.plus(amount);
```

Comment:
Please notice one validator can only be `epoch_unstake()` once, because the validator will be pending release if unstaked.